WooCommerce/src/main/kotlin/com/woocommerce/android/ui/main/MainActivityViewModel.kt

```diff
@@ -96,8 +96,8 @@ class MainActivityViewModel @Inject constructor(
         if (notification.channelType == NotificationChannelType.REVIEW) {
             triggerEvent(ViewReviewDetail(notification.uniqueId))
         } else if (notification.channelType == NotificationChannelType.NEW_ORDER) {
-            siteStore.getSiteBySiteId(notification.remoteSiteId)?.let { siteModel ->
-                triggerEvent(ViewOrderDetail(notification.uniqueId, siteModel.id, notification.remoteNoteId))
+            siteStore.getSiteBySiteId(notification.remoteSiteId)?.let {
+                triggerEvent(ViewOrderDetail(notification.uniqueId, notification.remoteNoteId))
```

Comment:
If I can see correctly, `getSiteBySiteId` is not used anymore, is that correct? Can we change it to something like

```suggestion
                triggerEvent(ViewOrderDetail(notification.uniqueId, notification.remoteNoteId))
```

Reply:
Good catch! Fixed here - https://github.com/woocommerce/woocommerce-android/pull/5518/commits/76d56b7763375da6f0bf5ff6a074d74db8e3ac7e

Reply:
Although, can it be that this is actually a valid case when the site is not available locally and then we just open the order list?

Reply:
Good point, it might be a valid case! So my suggestion would to still refactor this to not use `let` as it's confusing to use `let` without parameter.

Reply:
Good point - done 98d74d26fa2245a492431cbd5a3abef4e29c2f3c